app/uk/gov/hmrc/incometaxsoftwarechoicesfrontend/forms/FiltersForm.scala

```diff
@@ -16,27 +16,42 @@
 
 package uk.gov.hmrc.incometaxsoftwarechoicesfrontend.forms
 
-import play.api.data.Forms.{mapping, optional, text}
+import play.api.data.Forms.{list, mapping, optional, text}
 import play.api.data.validation.Constraint
 import play.api.data.{Form, Mapping}
 import uk.gov.hmrc.incometaxsoftwarechoicesfrontend.forms.utils.StringConstraints
-import uk.gov.hmrc.incometaxsoftwarechoicesfrontend.models.FiltersFormModel
+import uk.gov.hmrc.incometaxsoftwarechoicesfrontend.models.VendorFilter.{FreeTrial, FreeVersion, filterKeyToFilter}
+import uk.gov.hmrc.incometaxsoftwarechoicesfrontend.models.{FiltersFormModel, VendorFilter}
 
 object FiltersForm {
   val searchTerm = "searchTerm"
+  val filters = "filters"
+  val freeVersion = FreeVersion.key
```

Comment:
Any reason to expose these like this instead of just using the VendorFilter values directly in the view?